### PR TITLE
Add ocsms app

### PR DESCRIPTION
**Pull request by nerzhul**
@LukasReschke done

### CODE REVIEW

File: ocsms/ocsms [hunk 1]
```diff
@@ -0,0 +1,26 @@
+-----BEGIN CERTIFICATE REQUEST-----
```
Comment (nickvergessen) [inline ocsms/ocsms]: invalid filename

**Comment by nerzhul** [comment]
@nickvergessen fixed, sorry :)

**Comment by LukasReschke** [comment]
Certificate at https://github.com/nextcloud/app-certificate-requests/blob/master/ocsms/ocsms.crt

**Comment by nerzhul** [comment]
Thanks
it seems i'm unable to register. the form tell me it's complete but when i upload app upload is stuck to "upload in progress"

**Comment by MariusBluem** [comment]
> it seems i'm unable to register. the form tell me it's complete but when i upload app upload is stuck to "upload in progress"

This seems to be something for the appstore-repo: https://github.com/nextcloud/appstore 😉

**Comment by nerzhul** [comment]
@MariusBluem this is now solved :)

**Pull request merged**